tests/test_runners/test_pytest.py

```diff
@@ -49,6 +51,24 @@ def test_record_test_pytest(self):
         expected = self.load_json_from_file(self.result_file_path)
         self.assert_json_orderless_equal(expected, payload)
 
+    @responses.activate
+    @mock.patch.dict(os.environ, {"LAUNCHABLE_TOKEN": CliTestCase.launchable_token})
+    def test_record_test_with_json_option(self):
+        result = self.cli('record', 'tests',  '--session', self.session,
+                          'pytest', '--json', str(self.test_files_dir.joinpath("report.json")))
+
+        self.assertEqual(result.exit_code, 0)
+        payload = json.loads(gzip.decompress(
+            responses.calls[1].request.body).decode())
+        expected = self.load_json_from_file(self.result_file_path)
+
+        for e in payload["events"]:
+            del e["created_at"]
+        for e in expected["events"]:
+            del e["created_at"]
```

Comment:
[nits]
Better to use `e.pop("created_at", "")` than `del e["created_at"]` to remove a key from dict.
If there is no "created_at" in the dict, `del` will raise KeyError, while `pop` with secondary parameter will not raise any Error.